tests/UI/campaigns/functional/BO/09_shipping/02_preferences/02_carrierOptions/02_sortBy.js

```diff
@@ -0,0 +1,256 @@
+require('module-alias/register');
```

Comment:
I would suggest to maybe change the file name and all description to `updateCarriersSortOption`